sumo/cli/bandplot.py

```diff
@@ -306,9 +306,22 @@ def bandplot(filenames=None, code='vasp', prefix=None, directory=None,
             dos, pdos = load_dos(dos_file, elements, lm_orbitals, atoms, gaussian,
                                  total_only)
         elif code == 'castep':
-            dos = read_castep_tdos(dos_file, gaussian=gaussian,
-                                   efermi_to_vbm=True)
-            pdos = {}
+            pdos_file = None
+            if cell_file:
+                pdos_file = cell_file.replace('.cell', '.pdos_bin')
```

Comment:
What happens if the user has ".cell" in their seedname? (e.g. `prim.cell.2.cell, prim.cell.2.param`). Would be good to make this explicitly substitute the file extension to be safe.

Reply:
solved with more robust method for file extension substitution